tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/GoldParseTrainingVectorSource.scala

```diff
@@ -68,6 +68,9 @@ case class GoldParseTrainingVectorSource(
     for {
       goldParse <- goldParses.parseIterator
       vector <- generateVectors(goldParse)
-    } yield vector
+    } yield {
+      //println(vector)
```

Comment:
Looks like diagnostic print is commented out. Revert back to `yield vector` the way it was?